README.md

```diff
@@ -1,2 +1,31 @@
-# go-training
-Go programming language training
+# Go Training
+
+[![Build Status](https://travis-ci.com/anz-bank/go-training.svg?branch=master)](https://travis-ci.com/anz-bank/go-training)
+[![Coverage](https://codecov.io/gh/anz-bank/go-training/branch/master/graph/badge.svg)](https://codecov.io/gh/anz-bank/go-training)
+[![GolangCI](https://golangci.com/badges/github.com/anz-bank/go-training.svg)](https://golangci.com/r/github.com/anz-bank/go-training)
+
+This project is a playground for hands-on golang training.
+
+## Prerequisites
+
+-   Install `go 1.11` according to [official installation instruction](https://golang.org/doc/install)
+-   Clone this project outside your `$GOPATH` to enable [Go Modules](https://github.com/golang/go/wiki/Modules)
+-   Install `golangci-lint` according to [instructions](https://github.com/golangci/golangci-lint#local-installation)
+
+## Build, execute, test, lint
+
+Build and install this project with
+
+    go install ./...
+
+execute its binary with
+
+    0_hello_world
```

Comment:
I'm not sure where `go install ./...` installs a binary when using modules (no `GOPATH`), but if it is not on your `PATH`, then you will need some sort of path on this line, if only `./0_hello_world`. If it is on your `PATH`, you're assuming the user has their `PATH` set up the same way (`$GOPATH/bin` I assume).

Reply:
Do you know where go installs a project's dependencies if the `GOAPTH` isn't set? With`GOPATH` they seem to end up in `$GOPATH/pkg/mod`.
I think people keep thinking that `go mod` doesn't require `GOPATH` at all any more, but I'm not sure that that's true, it just means _your_ development doesn't have to happen in `$GOPATH/src/github.com/your/repo`any longer .... I might be mistaken though.